### PR TITLE
Fix squid-cache.org web paths

### PR DESCRIPTION
**Pull request by yadij**
Seems I tested the cfg manual script with a simpler webpath than
actually available on the server. Add the missing "content/" directory.

### CODE REVIEW

**Review by kinkie — APPROVED** [review]

File: release/mk-cfgman-docs.sh [hunk 1]
```diff
@@ -16,12 +16,12 @@ cd ${webpath}/content/Doc/config
 
 # Detect which v3 releases exist right now and grok them all.
 # TODO: ignore these versions as also no longer changing.
-VERS3=`ls -1t ${webpath}/Versions/v3/ | grep -v "CVS|sig.dyn"`
+VERS3=`ls -1t ${webpath}/content/Versions/v3/ | grep -v "CVS|sig.dyn"`
```
Comment (rousskov) [inline release/mk-cfgman-docs.sh:19]:
```suggestion
VERS3=`ls -1t ${webpath}/content/Versions/v3/ | grep -Ev "CVS|sig.dyn"`
```

File: release/mk-cfgman-docs.sh [hunk 2]
```diff
@@ -42,12 +42,12 @@ for v in ${VERS3} "$@" ; do
 done
 
 # Dynamically detect which v4+ releases exist right now and grok them all.
-VERS4=`ls -1t ${webpath}/Versions/ | grep -v "CVS|sig.dyn"`
+VERS4=`ls -1t ${webpath}/content/Versions/ | grep -v "CVS|sig.dyn"`
```
Comment (rousskov) [inline release/mk-cfgman-docs.sh:45]:
```suggestion
VERS4=`ls -1t ${webpath}/content/Versions/ | grep -Ev "CVS|sig.dyn"`
```

Comment (rousskov) [inline release/mk-cfgman-docs.sh:45]: FWIW, AFAICT, there are non-version files in that directory that are ignored by this "filter non-versions" `grep`:

```
langpack
index.dyn
makeweb.sh
authors.sh
```

They will be skipped due to the `test -d` below, but so is `sig.dyn` file present in this `grep` regex. Thus, even if we ignore the fact that files like `auhtors.sh` are not "versions", there is an inconsistency here.

**Review by rousskov — APPROVED** [review]
LGTM. I found two out-of-scope bugs and one inconsistency, but they do not need to be addressed in this PR (or at all if this is legacy, barely used code).

**Comment by yadij** [comment]
> LGTM. I found two out-of-scope bugs and one inconsistency, but they do not need to be addressed in this PR (or at all if this is legacy, barely used code).

Improvement PRs are welcome.

**Pull request merged**